src/Foundatio.Kafka/Messaging/KafkaMessageBusOptions.cs

```diff
@@ -10,17 +10,15 @@ public class KafkaMessageBusOptions : SharedMessageBusOptions {
 
     public string GroupId { get; set; }
 
+    public string MessageType { get; set; } = "MessageType";
+
     public string ContentType { get; set; } = "text/json";
 
-    // TODO: Ensure builder
     public string PublishKey { get; set; }
 
-    public int AutoCommitIntervalMs { get; set; } = 5000;
-
-    // TODO: Ensure we are setting this on kafka lib config
-    public AutoOffsetReset AutoOffSetReset { get; set; }
+    public Func<ConsumeResult<string, byte[]>, string> ResolveMessageType { get; set; }
```

Comment:
I don't think this is being used, can you check the implementation to resolve this.

```cs
      try {
            string messageType = _options.ResolveMessageType?.Invoke(consumeResult);
            if (String.IsNullOrEmpty(messageType)) {
                var messageTypeHeader = consumeResult.Message.Headers.SingleOrDefault(x => x.Key.Equals(MessageType));
                if (messageTypeHeader != null)
                    messageType = Encoding.UTF8.GetString(messageTypeHeader.GetValueBytes());
            }
            
            // What to do if message type is null?
            var message = ConvertToMessage(messageType, consumeResult.Message.Value);
            await SendMessageToSubscribersAsync(message).AnyContext();
        } catch (Exception ex) {
            _logger.LogWarning(ex, "OnMessage({Offset}] {Partition}) Error deserializing message: {Message}", consumeResult.Offset, consumeResult.TopicPartition.Partition, ex.Message);
        }
```